datastore/integration_test.go

```diff
@@ -225,6 +225,9 @@ func TestIntegration_GetWithReadTime(t *testing.T) {
 		t.Fatalf("Transaction.Commit: %v\n", err)
 	}
 
+	// We need to wait a few seconds to allow eventual consistency to catch up.
+	time.Sleep(time.Duration(10 * math.Pow(10, 9)))
```

Comment:
This time is not really readable, nor the line below. Please do something like:

```
time.Sleep(10 * time.Second)
```

Also, we are are already retrying, why do we need to sleep as well? The retry will sleep on failure, right?

Reply:
(Thanks for the tip about expressing seconds :)! I had forgotten about that constant.)

It seems that we need more time for the database to catch up with the test. It's hard for me to determine what is the correct fix here, since my local environment differs so much from the integration environment.

I'll try to increase the sleep on the retry to see if that fixes this issue and delete this first call to sleep.